google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java

```diff
@@ -1057,6 +1076,7 @@ public HttpResponse execute() throws IOException {
         }
       }
     } while (retryRequest);
+    span.end(OpenCensusUtils.getEndSpanOptions(response == null ? null : response.getStatusCode()));
```

Comment:
Before ending the span is good to record a MessageEvent with the received message size.